web/tests/message_list.test.js

```diff
@@ -359,19 +359,20 @@ run_test("bookend", ({override}) => {
 
     override(stream_data, "is_subscribed", () => is_subscribed);
     override(stream_data, "get_sub_by_id", () => ({invite_only, name: "IceCream"}));
-    override(stream_data, "can_toggle_subscription", () => true);
```

Comment:
These changes should be squashed into the commit that makes the test fail.

Check out our [GitHub guide](https://zulip.readthedocs.io/en/latest/git-guide.html) and [commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html) for more details.

Reply:
done, now each commit passes all tests individually. I’ll keep this in mind for future PRs.